src/game_battler.cpp

```diff
@@ -427,8 +427,16 @@ bool Game_Battler::AddState(int state_id, bool allow_battle_states) {
 	return was_added;
 }
 
-bool Game_Battler::RemoveState(int state_id) {
-	auto was_removed = State::Remove(state_id, GetStates(), GetPermanentStates());
+bool Game_Battler::RemoveState(int state_id, bool always_battle_states) {
```

Comment:
>  RemoveState event commands remove battle equip states 

Could you give the var a better name, like ``[always_]remove_battle_states``

Reply:
Fixed